libr/libs.mk

```diff
@@ -4,7 +4,7 @@ ifeq ($(LIBS0),)
 LIBS0=util
 LIBS1=socket hash reg cons magic bp search config
 LIBS2=syscall lang io crypto flag
-LIBS3=fs anal bin parse
+LIBS3=fs parse anal bin
```

Comment:
Those 4 libs are built in paralel. If anal depends on parse must be moved into libs4